src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -787,10 +787,15 @@ def _find_missing_deps(self, compile_failure_log, target):
         for classname, candidates in missing_dep_suggestions.items():
           suggested_deps.add(list(candidates)[0])
           self.context.log.info('  {}: {}'.format(classname, ', '.join(candidates)))
+
+        # We format the suggested deps with single quotes and commas so that
+        # they can be easily cut/pasted into a BUILD file.
+        formatted_suggested_deps = ["'%s'," % dep for dep in suggested_deps]  
         suggestion_msg = (
           '\nIf the above information is correct, '
-          'please add the following to the dependencies of ({}):\n  {}\n'
-            .format(target.address.spec, '\n  '.join(sorted(list(suggested_deps))))
+          'please add the following to the dependencies of ({}):\n  {}\n\n'
+          'Remember to sort the dependencies in your BUILD file for easier maintenance.\n'
```

Comment:
This line about sorting is a bit orthogonal. Would rather it was dropped, as I don't think its a thing that's worth having humans do until it is automated.